ert3/engine/_record.py

```diff
@@ -7,10 +7,8 @@
 import yaml
 
 
-def load_record(workspace: Path, record_name: str, record_file: TextIO) -> None:
-    raw_ensrecord = json.load(record_file)
-
-    record_file.close()
```

Comment:
Should we not close the files outside of this method then? In the test for examples the files are never closed 🤔